### ISSUE TITLE
libraries should work with range of dependency versions

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/commons#34]
Trying to resolve https://github.com/hubmapconsortium/portal-ui/pull/1403 -- pip install causes this error:
```
hubmap-commons 1.12.7 has requirement PyYAML==5.3.1, but you'll have pyyaml 5.2 which is incompatible.
```

As a general principle, python libraries should try to work with a range of dependency versions, but setup.py specifies:
```
                      'neo4j==1.7.2',
                      'jsonschema==3.2.0',
                      'requests==2.22.0',
                      'jsonref-ap==0.3.dev0',
                      'PyYAML==5.3.1'
```

**Comment by yuanzhou** [comment]
Issue has been addressed in the current `test-release` branch across commons, gateway, ingest-api, uuid-api, entity-api, and search-api. Now in their `requirements.txt`, they all use the same versions:
````
# It's an agreement with other collaborators to use the below versions
# for requests and PyYAML
requests>=2.22.0
PyYAML>=5.3.1
````

@mccalluc @jswelling please make the corresponding updates in the ingest-validation-tools.

**Comment by mccalluc** [comment]
@yuanzhou : repeating what I said I slack: We need a new release of commons, to replace 
```
git+git://github.com/hubmapconsortium/commons.git@v1.12.7#egg=hubmap-commons
```

**Comment by yuanzhou** [comment]
@mccalluc just saw your slack message. 

I just released the v2.0.0 of commons on github. You can just use
````
git+git://github.com/hubmapconsortium/commons.git@master#egg=hubmap-commons
````
instead of the version tag. This way you don't need to worry about future change. On our end, the master branch code is ALWAYS the latest released version corresponding the latest tag.
